tests/unittest/PinAssignment/PackagePinsLoader_test.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2022 The Foedag team
+
+GPL License
+
+Copyright (c) 2022 The Open-Source FPGA Foundation
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "PinAssignment/PackagePinsLoader.h"
+
+#include <filesystem>
+
+#include "PinAssignment/PackagePinsModel.h"
+#include "gtest/gtest.h"
+using namespace FOEDAG;
+
+TEST(PackagePinsLoader, load_general) {
```

Comment:
Please follow the naming convention for gtests.
_When naming your test suites and tests, you should follow the same convention as for [naming functions and classes](https://google.github.io/styleguide/cppguide.html#Function_Names)._
Why ? 
https://google.github.io/googletest/faq.html#why-should-test-suite-names-and-test-names-not-contain-underscore

Reply:
this is done.

Reply:
Some tests are still with camelCase as against CamelCase.

Reply:
I prefer to see test suit from upper letter like class name and test name from lower case like class method. For me it is more natural. But lets define some documentation on that.

Reply:
It's about following the gtest guidelines (defined in their docs already, we don't need a separate one) for naming conventions. Unless we have a good reason to deviate from it we should stick follow them for others who will look at these tests later.

Reply:
Done.